Update reference documentation (add data model and GeoPackage) (#310)
Open-Telecoms-Data/open-fibre-data-standard: docs/reference/publication_formats/index.md

To comment on or create discussions and issues, you need to [sign up for a free GitHub account](https://github.com/signup). If you prefer to provide feedback privately, you can email [info@opentelecomdata.net](mailto:info@opentelecomdata.net).
```

> OFDS defines standardised representations of the [logical data model](../data_model.md) in the following data formats:

Reviewer: ```suggestion
OFDS defines standardised implementations of the [logical data model](../data_model.md) for the following data formats:
```

I think it's more helpful to talk about 'implementations' of a data model (so that we can talk about how we 'represent' it diagrammatically for example).

Author: Yes, 'representations' might be a bit too general. However, I'd prefer to avoid using "implementation" in this context, as I think it'll cause confusion with the typical meaning of "implementation" in a data standard context being an adopter's use of the standard.

How about:

```suggestion
OFDS defines schemas based on the [logical data model](../data_model.md) for publishing, storing and exchanging data in the following formats:
```

Reviewer: Yes - your edit looks good. (I'll leave you to commit it and resolve this convo.)
